kernel/classes/datatypes/ezobjectrelationlist/ezobjectrelationlisttype.php

```diff
@@ -76,7 +76,8 @@ function validateObjectAttributeHTTPInput( $http, $base, $contentObjectAttribute
             // If in browse mode and relations have been added using the search field
             // items are stored in the post variable
             if (
-                $http->postVariable( $postVariableName ) != array( "no_relation" )
+                is_array( $http->postVariable( $postVariableName ) )
```

Comment:
If it's null I guess it's not set, so then more correct would be to make sure we check if it's here:
```suggestion
                $http->hasPostVariable( $postVariableName )
```

Reply:
Yes, that seems reasonable.